docs/ref/models/expressions.txt

```diff
@@ -228,6 +228,22 @@ primary key value rather than a model instance::
     >> car.built_by
     3
 
+.. _using-f-to-sort-nulls-last:
+
+Using ``F()`` to sort nulls last
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``F()`` can be used to sort a model field with nulls last.
+The code below will sort the `last_contacted` nullable datetime field with nulls first::
+
+    companies = Company.objects.order_by('-last_contacted')
+
+However, if you want nulls to be sorted last, such that those companies that have been contacted would be sorted first, you can use ``F()`` expressions::
```

Comment:
Please wrap lines at 79 characters.

Reply:
"F() expressions and Expression.asc() / desc()" (with links for the latter items)